service.go

```diff
@@ -86,9 +86,14 @@ func (s *Service) SearchCerts(clusterID string) (AssetsBundle, error) {
 // SearchCertsForComponent watches for secrets of a single cluster component and
 // returns it as assets bundle.
 func (s *Service) SearchCertsForComponent(clusterID, componentName string) (AssetsBundle, error) {
+	return s.SearchCertsForComponentInNamespace(api.NamespaceDefault, clusterID, componentName)
```

Comment:
I decided that it makes sense just to call SearchCertsForComponentInNamespace

Reply:
I think it makes sense to replace SearchCertsForComponent with SearchCertsForComponentInNamespace and remove SearchCertsForComponentInNamespace

Reply:
Mr.Schindler recommended to keep `SearchCertsForComponent` for compatibility.

Reply:
Yeah, we should start with thinking about how to change our code base and how to move forward without breaking existing implementations. There should also be plans to deprecate functionality and a smooth migration path even for ourselves. This will be quite a challenge and we have to start somewhere. I think this will be one of the bigger challenges for ourselves in the upcoming months.

Reply:
I'm cool with that. Go ahead.